context/app/static/js/components/DetailMetadata.jsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable camelcase */
+import React from 'react';
+import styled from 'styled-components';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import SectionHeader from './SectionHeader';
+import SectionItem from './SectionItem';
+import SectionContainer from './SectionContainer';
+
+const StyledTypography = styled(Typography)`
+  margin: 2px 0px 2px 0px;
+`;
+
+const FlexPaper = styled(Paper)`
+  display: flex;
+  padding: 30px 40px 30px 40px;
+`;
+
+function MetadataItem(props) {
+  const { label, value, ml } = props;
+  return (
+    <SectionItem label={label} ml={ml}>
+      <StyledTypography variant="body1">
+        {value || `${label} not defined`}
+      </StyledTypography>
+    </SectionItem>
+  );
+}
+
+function DonorItems(props) {
+  const { metadata } = props;
+
+  return (
+    <>
+      <MetadataItem label="Gender Finding" value={metadata['Gender finding']} />
```

Comment:
Add a TODO note here... This will change when we have this information in a facetable shape.

Reply:
Done